src/cache_cf.cc

```diff
@@ -2999,7 +2998,7 @@ dump_time_msec(StoreEntry * entry, const char *name, time_msec_t var)
 void
 parse_time_msec(time_msec_t * var)
```

Comment:
Please fix this API as well.

Reply:
This is out of scope for this PR - no callers of this API suffer from initialization timing issues. They are all initialized to a preset default long before the call. I will do a followup refactor PR instead.